examples/Libraries/grafica/LifeExpectancy/LifeExpectancy.rpde

```diff
@@ -0,0 +1,58 @@
+# Load the table before the sketch is run
```

Comment:
Should this comment appear above line 10 instead?

Reply:
Yeah it is a mistake.